spec/acceptance/basic_posix_acl_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'posix_acl' do
+  context 'on a simple acl with an additional user' do
+    it_behaves_like 'an idempotent resource' do
+      let(:manifest) do
+        <<-EOS
+        include posix_acl::requirements
+        user { 'blub':}
+        file { '/opt/test':
+          ensure => directory,
+          owner  => root,
+          group  => root,
+          mode   => '2770',
+        }
+        -> posix_acl { '/opt/test':
+          action     => exact,
+          permission => [
+            'user::rwx',
+            'group::rwx',
+            'mask::rwx',
+            'other::---',
+            'user:blub:r-x',
+          ],
+          provider   => posixacl,
+          recursive  => false,
+        }
+        EOS
+      end
+    end
+  end
+
+  context 'on a simple acl with an additional user and defaults' do
+    it_behaves_like 'an idempotent resource' do
+      let(:manifest) do
+        <<-EOS
```

Comment:
I think `EOS` is frowned on by rubocop. That's why I typically use `PUPPET` instead, though there is technically no strict requirement on that.